composeApp/src/commonMain/kotlin/common/Modifiers.kt

```diff
@@ -0,0 +1,16 @@
+package common
+
+import androidx.compose.ui.Modifier
+
+fun Modifier.conditional(
+    condition: Boolean,
+     ifModifier: Modifier): Modifier {
+    return if (condition) ifModifier else Modifier
+}
+
+fun Modifier.conditionalWithElse(
```

Comment:
This modifiers are not being used. Please remove them.

Reply:
Will remove.